docs/developers/CII-BestPractices-status.md

```diff
@@ -467,50 +474,49 @@ usually to address use of 3rd party library headers that trigger warnings.
 
 </ul>
 
-
 <b><i>Use basic good cryptographic practices</i></b>
```

Comment:
I would suggest "**N/A**" for all the security topics.

Reply:
OCIO reads some external files introducing some security concern to any apps using it. Please forget my comment around security topics as it partially applies.